app/app_main.c

```diff
@@ -51,8 +51,15 @@
 #include <openssl/bn.h>
 #include <openssl/rand.h>
 #include <openssl/kdf.h>
+// #include <openssl/fipssyms.h>
+#include <openssl/fips_rand.h>
+#include <openssl/fips.h>
+#include "app_lcl.h"
 
-#ifdef ACVP_NO_RUNTIME
+// #include <openssl/fips.h>
```

Comment:
Same here, is this needed ? Or is it something we need to reconcile regarding building with CiscoSSL/FOM versus building with just CiscoSSL/openssl ?